pkg/framework/aries/factory/transport/transport_factory.go

```diff
@@ -21,7 +24,6 @@ func NewProviderFactory() *ProviderFactory {
 }
 
 // CreateOutboundTransport returns a new default implementation of outbound transport provider
-func (f *ProviderFactory) CreateOutboundTransport() transport.OutboundTransport {
-	// TODO - https://github.com/hyperledger/aries-framework-go/issues/83
-	return nil
+func (f *ProviderFactory) CreateOutboundTransport() (transport.OutboundTransport, error) {
+	return httptransport.NewOutbound(httptransport.WithOutboundHTTPClient(http.DefaultClient))
```

Comment:
Should we be using http.DefaultClient?

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/d38072ca1e0f1af9e3eb924c40eb43fee3575b9f..cf833874d8521c254c109468b96267de58218af9) to create a new client.